### PR TITLE
Fix/model comments

### PR DESCRIPTION
**Pull request by menma331**
All models docstring has been changed to more understandable style.

### CODE REVIEW

File: src/apps/comments/models.py
Comment (mtedoradze) [inline src/apps/comments/models.py:1]: Я бы добавила везде артикль "a" к словам, обозначающим объекты и не имеющим определений (напр., "author of a comment"),  и немного уточнила некоторые формулировки - см. ниже комментарии по тексту :)

File: src/apps/comments/models.py [hunk 1]
```diff
@@ -8,7 +8,16 @@
...
+
+    Attrs:
+        • author (ForeignKey): author of comment.
+        • recipe (ForeignKey): recipe of comment.
+        • text (TextField): text of comment.
+        • pub_date (DateTimeField): date of publish comment.
+        • parent (ForeignKey): answer for comment.
+        • reactions (GenericRelation): reaction for comment.
+        • updated_date (DateTimeField): date of updating comment.
     """
```
Comment (mtedoradze) [inline src/apps/comments/models.py:21]: * recipe (ForeingKey): recipe, on which a comment was made.
* pub_date (DateTimeField): comment publication date.
* parent (ForeignKey): initial comment, on which a comment was made.
* updated_date (DateTimeField): comment update date.

File: src/apps/favorite/models.py [hunk 2]
```diff
@@ -5,6 +5,11 @@
 class Favorite(models.Model):
     """
     Favorite model
+
+    Attrs:
+    • author (ForeignKey): author of favorite recipe.
+    • recipe (ForeignKey): recipe that has been added to favorite.
+    • pub_date (DateTimeField): date that recipe has been added to favorite.
     """
```
Comment (mtedoradze) [inline src/apps/favorite/models.py:14]: поменяй "added to favorite" на "added to favorites"

File: src/apps/ingredients/models.py [hunk 3]
```diff
@@ -29,6 +36,12 @@ def __str__(self):
 class IngredientInRecipe(models.Model):
     """
     IngredientInRecipe model
+
+    Attrs:
+    • ingredient (ForeignKey): ingredient in recipe.
+    • recipe (ForeignKey): recipe of ingredient.
+    • unit (ForeignKey): unit of ingredient in recipe.
+    • amount (PositiveIntegerField): amount of ingredients.
     """
```
Comment (mtedoradze) [inline src/apps/ingredients/models.py:45]: • recipe (ForeignKey): recipe, in which an ingredient is added.
 • amount (PositiveIntegerField): amount of an ingredient.

File: src/apps/reactions/models.py [hunk 4]
```diff
@@ -9,6 +9,16 @@
...
+    Attrs:
+    • author (ForeignKey): author of reaction.
+    • is_deleted (BooleanField):  reaction is deleted ?
+    • pub_date (DateTimeField): date of publish reaction.
+    • emoji (CharField(choices)): emoji of reaction.
+    • limit_models (Q): comment or recipe.
+    • content_type (ForeignKey): date of updating comment.
+    • object_id (PositiveIntegerField): id of object.
+    • reaction_object (GenericForeignKey(content_type, object_id)): object of reaction.
     """
```
Comment (mtedoradze) [inline src/apps/reactions/models.py:24]: • is_deleted (BooleanField): indicates whether reaction was deleted by it's author
• pub_date (DateTimeField): reaction publication date.
• content_type (ForeignKey): indicates a model of an object on which a reaction was made.
• object_id (PositiveIntegerField): id of an object on which a reaction was made.
• reaction_object (GenericForeignKey(content_type, object_id)): indicates an object on which a reaction was made.

File: src/apps/recipes/models.py [hunk 5]
```diff
@@ -16,6 +16,23 @@
...
+    • ingredients (ManyToManyField): ingredients of recipe.
+    • tag (TaggableManager): tag of recipe.
+    • category (ManyToManyField): category of recipe.
+    • cooking_time (PositiveIntegerField): time of cooking.
+    • pub_date (DateTimeField): recipe publication date.
+    • updated_at (DateTimeField): date of updating recipe.
+    • reactions (GenericRelation): reactions for recipe.
+    • is_repost (BooleanField): is repost. Default False.
+
     """
```
Comment (mtedoradze) [inline src/apps/recipes/models.py:36]: • full_text (TextField): recipe's full text.
• short_text (CharField(200)): recipe's short text.
• updated_at (DateTimeField): recipe updated date.
• reactions (GenericRelation): reactions on a recipe.
• is_repost (BooleanField): indicates whether recipe was reposted. Default False.

File: src/apps/users/models.py [hunk 6]
```diff
@@ -55,6 +55,22 @@ def create_superuser(self, email, username=None, password=None, **extra_fields):
...
+    • city (CharField(30)): city of user.
+    • first_name (CharField(30)): user first name.
+    • last_name (CharField(30)): user last name.
+    • bio (CharField(200)): user bio.
+    • avatar (ImageField): user avatar.
+    • is_banned (BooleanField(default=False)): user is banned.
+    • is_active (BooleanField(default=True)): user is active.
+    • is_staff (BooleanField(default=False)): user is staff.
+    • is_admin (BooleanField(default=False)): user is admin.
     """
```
Comment (mtedoradze) [inline src/apps/users/models.py:76]: Это надо удалить: "pub_date (DateTimeField): date of publish comment." - по ошибке, наверно, попало сюда :)

• display_name (CharField): user's name to be displayed. 
• is_banned (BooleanField(default=False)): indicates whether user is banned.
• is_active (BooleanField(default=True)): indicates whether user is active.
• is_staff (BooleanField(default=False)): indicates whether  user is staff.
• is_admin (BooleanField(default=False)): indicates whether user is admin.

File: src/apps/comments/models.py (hunk 1 above)
Comment (menma331) [reply src/apps/comments/models.py:21]: Решил. Запушил.

**Review by vBaMnup — APPROVED** [review]

**Review by mtedoradze — APPROVED** [review]

**Pull request merged**